src/game/game.gd

```diff
@@ -0,0 +1,19 @@
+extends Node2D
+
+onready var mapNode: Node2D = $Map
+onready var characterNode: Node2D = $Characters
+onready var rng = RandomNumberGenerator.new()
+
+var characterTemplate: Resource = ResourceLoader.load("res://game/character/character.tscn")
+
+func loadMap(mapPath: String) -> void:
+	var mapToLoad: Node = ResourceLoader.load(mapPath).instance()
+	add_child_below_node(mapNode, mapToLoad)
+
+func addCharacter() -> void:
+	var newCharacter: KinematicBody2D = characterTemplate.instance()
+	var position: Vector2
+	position.x = rng.randi_range(100, 500)
+	position.y = rng.randi_range(100, 500)
+	add_child_below_node(characterNode, newCharacter)
+	newCharacter.setPosition(position)
```

Comment:
In addition to the last note about using the Characters autoload for stuff, this set position should ideally be going through the character resource first, so it'd look like 
```swift
newCharacterResource.setPosition(position)
```

Reply:
done.